roseus/package.xml

```diff
@@ -41,6 +41,7 @@
   <build_depend>dynamic_reconfigure</build_depend>
   <build_depend>actionlib_tutorials</build_depend>
   <build_depend>coreutils</build_depend>
+  <build_depend>jsk_recognition_msgs</build_depend>
```

Comment:
is it `build_depend`?

Reply:
Thank you for your review.
I removed this line.